docs/src/examples/components/Tooltip/Types/TooltipExamplePointing.shorthand.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react'
+import { Button, Grid, Tooltip } from '@stardust-ui/react'
+
+const ButtonWithTooltip = props => (
+  <Tooltip
+    align={props.align}
+    content={{
+      content: (
+        <>
+          <p>A tooltip with a pointer.</p>
+          <p>
+            Is aligned to <code>{props.align}</code>.
+          </p>
+        </>
+      ),
+    }}
+    pointing
+    position={props.position}
+    trigger={
+      <Button
+        icon={props.icon}
+        styles={{ padding: props.padding, height: '38px', minWidth: '64px' }}
+      />
+    }
+  />
+)
+
+const triggers = [
+  { position: 'above', align: 'start', icon: 'arrow circle up', padding: '5px 42px 18px 5px' },
+  { position: 'above', align: 'center', icon: 'arrow circle up', padding: '5px 5px 18px 5px' },
+  { position: 'above', align: 'end', icon: 'arrow circle up', padding: '5px 5px 18px 42px' },
+  { position: 'below', align: 'start', icon: 'arrow circle down', padding: '18px 42px 5px 5px' },
+  { position: 'below', align: 'center', icon: 'arrow circle down', padding: '18px 5px 5px 5px' },
+  { position: 'below', align: 'end', icon: 'arrow circle down', padding: '18px 5px 5px 42px' },
+  { position: 'before', align: 'top', icon: 'arrow circle left', padding: '5px 42px 18px 5px' },
+  { position: 'before', align: 'center', icon: 'arrow circle left', padding: '5px 42px 5px 5px' },
+  { position: 'before', align: 'bottom', icon: 'arrow circle left', padding: '18px 42px 5px 5px' },
+  { position: 'after', align: 'top', icon: 'arrow circle right', padding: '5px 5px 18px 42px' },
+  { position: 'after', align: 'center', icon: 'arrow circle right', padding: '5px 5px 5px 42px' },
+  { position: 'after', align: 'bottom', icon: 'arrow circle right', padding: '18px 5px 5px 42px' },
+]
```

Comment:
Can we add `pointing` prop into position example instead and keep only simple example there?

Reply:
Yep, added.